### PR TITLE
fixed the prow e2e test issues

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>
Fixed: https://github.com/stolostron/backlog/issues/24257

### CODE REVIEW

File: test/setup/e2e_dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  echo "docker version: $(docker version --format '{{.Client.Version}}')"
+}
+
+function checkVolume() {
+  if [[ $(df -h | grep -v Size | awk '{print $2}' | sed -e 's/G//g' | awk 'BEGIN{ max = 0 } {if ($1 > max) max = $1; fi} END{print max}') -lt 80 ]]; then
+    maxSize=$(lsblk | awk '{print $1,$4}' | grep -v Size | grep -v M | sed -e 's/G//g' | awk 'BEGIN{ max = 0 } {if ($2 > max) max = $2; fi} END{print max}')
+    mountName=$(lsblk | grep "$maxSize"G | awk '{print $1}')
+    echo "mounting /dev/${mountName}: ${maxSize}"
+    sudo mkfs -t xfs /dev/${mountName}        
+    sudo mkdir /data/docker
```
Comment (clyang82) [inline test/setup/e2e_dependencies.sh:102]: replace `mkdir /data/docker` with `mkdir  -p /data/docker`

**Comment by yanmxa** [comment]
@clyang82 @morvencao /cc

**Comment by clyang82** [comment]
💯 Great Job!
/lgtm

**Pull request merged**